tsl/src/continuous_aggs/repair.c

```diff
@@ -362,7 +370,8 @@ get_direct_view_oid(int32 mat_hypertable_id)
 	relation_close(cagg_idx_rel, AccessShareLock);
 
 	/* Get Oid of user view */
-	Oid direct_view_oid = ts_get_relation_relid(view_schema, view_name, false);
+	Oid direct_view_oid =
+		ts_get_relation_relid(NameStr(view_schema_name), NameStr(view_name_name), false);
```

Comment:
So it takes char * but the actual object must be Name? This is a very bad interface, let's fix it as well?

Reply:
Yeahhh it is... but note that in Postgres it's also a bad interface. For example `get_namespace_oid` also have the `char *` as parameter but internally during syscache calls it will rely on `namehashfast` that will do `NameStr(*DatumGetName(datum))` ... :-1:

Reply:
I will merge this PR now to fix the failing CI. 

We could change the signature of `ts_get_relation_relid`. However, the PostgreSQL interface of the internally called functions is `char *`; it might make sense to use the same signature to be compatible and be able to pass the same elements to the function. 

If we decide to refactor this, we might need to change a couple of internal functions since they return `char *`. This is something for a follow-up PR.